docs/user-guide/cli-scsplugin.md

```diff
@@ -24,7 +24,7 @@ For detailed command, actions, and option documentation for this plug-in, see ou
 
 Before you install the plug-in, meet the software requirements in [Software requirements for Zowe CLI plug-ins](cli-swreqplugins.md).
 
-**Note:** If you're using npm@7, an Internet connection is required during installation of the Secure Credential Store plug-in. Additionally, there is a prerequisite on Windows for npm@7 users: `npm install -g prebuild-install`
+**Note:** If you're using npm@7 or npm@8, an Internet connection is required during installation of the Secure Credential Store plug-in. Additionally, there is a prerequisite on Windows for npm@7 and npm@8 users: `npm install -g prebuild-install`
```

Comment:
Not sure, why this is arranged as a note. Maybe you should formulate it as part of the pre-req. 

```suggestion
(npm@7 or npm@8 only) Ensure that you have an internet connection when you install the Secure Credential Store plug-in. If Windows requires npm@7 and npm@8, issue the following command:

```
`npm install -g prebuild-install`
```
```